src/pip/_internal/commands/cache.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import absolute_import
+
+import logging
+import os
+import textwrap
+
+import pip._internal.utils.filesystem as filesystem
+from pip._internal.cli.base_command import Command
+from pip._internal.cli.status_codes import ERROR, SUCCESS
+from pip._internal.exceptions import CommandError, PipError
+from pip._internal.utils.typing import MYPY_CHECK_RUNNING
+
+if MYPY_CHECK_RUNNING:
+    from optparse import Values
+    from typing import Any, List
+
+
+logger = logging.getLogger(__name__)
+
+
+class CacheCommand(Command):
+    """
+        Inspect and manage pip's caches.
```

Comment:
Here and some other places it's plural, others it's singular. Perhaps be consistent and stick with singular "cache"? Or is it referring to different types of cache?

Reply:
I'm not entirely sure why I was so inconsistent about it, but I'm changing it to always be singular.